news/_posts/core-weekly/2019-01-04-coreweekly-week-51-2018.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 51 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-01-04 16:15:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 17th to Sunday 23rd of December 2018.
```

Comment:
this is not the last week, maybe can you publish a "weekly report" for 2 consecutive weeks at the same time?